src/app/backend/handler/apihandler.go

```diff
@@ -1442,7 +1494,7 @@ func (apiHandler *APIHandler) handleGetPersistentVolumeList(request *restful.Req
 func (apiHandler *APIHandler) handleGetThirdPartyResource(request *restful.Request,
 	response *restful.Response) {
 	dataSelect := parseDataSelectPathParameter(request)
-	result, err := thirdpartyresource.GetThirdPartyResourceList(apiHandler.client, dataSelect)
+	result, err := thirdpartyresource.GetThirdPartyResourceList(request.Attribute("apiclient").(*clientK8s.Clientset), dataSelect)
```

Comment:
It will but it's better to do cast with error check so we can adjust the error message. This way it will be more clear for devs and also we will have single method where error can occur.